test/contracts/staking/staking-mediator.test.ts

```diff
@@ -130,7 +131,7 @@ describe('Staking Pool Mediator contract', () => {
                 const expectedDaoId = previousHighestDaoId.add(1).toBigInt()
```

Comment:
issue: You need t least one new test case: when `METADATA` is  `undefined`.

_Depends on what the expected behaviour is for metadata, whether it's mandatory (where you'll need more) or optional_

Reply:
Ok i've added one - `set as empty string`. Is this what you mean or you are talking about calling `createDao()` with an empty string?

Reply:
Yes, invoking `createDao()` with an empty string.